swagger-ui/startchain.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+# TODO import from development environment in envkey
+password="password"
+validatorMnemonic="equip town gesture square tomorrow volume nephew minute witness beef rich gadget actress egg sing secret pole winter alarm law today check violin uncover"
+# kava15qdefkmwswysgg4qxgqpqr35k3m49pkx2jdfnw
+
+#   address: kava1ffv7nhd3z6sych2qpqkk03ec6hzkmufy0r2s4c
+#   address: kavavaloper1ffv7nhd3z6sych2qpqkk03ec6hzkmufyz4scd0
+
+faucet="chief access utility giant burger winner jar false naive mobile often perfect advice village love enroll embark bacon under flock harbor render father since"
+# kava1xy7hrjy9r0algz9w3gzm8u6mrpq97kwta747gj
+
+# address: kava1ls82zzghsx0exkpr52m8vht5jqs3un0ceysshz
+# address: kavavaloper1ls82zzghsx0exkpr52m8vht5jqs3un0c5j2c04
+
+# Remove any existing data directory
+rm -rf ~/.kvd
```

Comment:
Avoid deleting the default installation of kvd and kvcli by using the --home flag with `kvd init`

Something like 
`kvd init --chain-id --home /tmp`
I believe you have to add the home flag to all additional  kvd commands

Reply:
ok I will address

Reply:
ok it has been updated now to use a temporary directory instead of the default installation